src/plot/plot.jl

```diff
@@ -534,15 +535,43 @@ function solution(
     clusters!(
         ax,
         soln.cluster_sets[end];
-        cluster_radius=cluster_radius,
-        nodes=true,
-        centers=false,
-        labels=true
+        labels=true,
+        cluster_radius
+    )
+
+    # Annotate critical path cost
+    vessel_weightings = (problem.mothership.weighting, problem.tenders.weighting)
+    critical_path_cost = critical_path(soln, vessel_weightings)
```

Comment:
The optimization is using `critical_distance_path` - did you want to show that instead?

Reply:
As discussed, the answer to the above is no.